<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Offset" name="offset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="Usb_mmf" name="usb_mmf">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>361</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="edit_secondLeg">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_max_secondLeg">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>90</string>
    </property>
   </widget>
   <widget class="QLabel" name="edit_thirdLeg">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="edit_firstLeg">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>306</width>
      <height>285</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="cb_timer">
         <property name="text">
          <string>&amp;Timer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="tbr_delay">
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Edit_timerDelay">
         <property name="placeholderText">
          <string>1000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="lv_commands"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="edPath"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_commands">
             <property name="text">
              <string>&amp;Load commands</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnClear">
             <property name="text">
              <string>&amp;Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="edit_min_firstleg">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-80</string>
    </property>
   </widget>
   <widget class="QSlider" name="tbr_thirdLeg">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="tbr_firstLeg">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_max_thirdLeg">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>90</string>
    </property>
   </widget>
   <widget class="QSlider" name="tbr_secondLeg">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_send">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lv_legs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_sign">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>offset 부호변경</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_offset">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>offset 적용해제</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_min_thirdLeg">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-90</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_min_secondleg">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_max_firstLeg">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>80</string>
    </property>
   </widget>
   <widget class="Usb_log" name="usb_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>631</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_firstLeg">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>[&amp;1] FirstLeg</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_secondLeg">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[&amp;2] SecondLeg</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_thirdLeg">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[&amp;3]ThirdLeg</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_firstSerial">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_secondSerial">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial 2</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lw_legs">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>141</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>805</x>
      <y>320</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QListWidget" name="lw_motors"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Set ids</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tempSave">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpen_App_Folder"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen_App_Folder">
   <property name="text">
    <string>Open App Folder</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Offset</class>
   <extends>QWidget</extends>
   <header>offset.hpp</header>
  </customwidget>
  <customwidget>
   <class>Usb_log</class>
   <extends>QWidget</extends>
   <header>usb_log.h</header>
  </customwidget>
  <customwidget>
   <class>Usb_mmf</class>
   <extends>QWidget</extends>
   <header>usb_mmf.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
